PYME/config.py

```diff
@@ -212,6 +212,8 @@
     to. Increasing the chunksize can increase data-locality for faster analysis,
     but has spooling/writing bandwidth implications.
 
+zoom_factor : default = 1.1, adjusts zoom sensitivity by adjusting magnification factor per scroll event
```

Comment:
We need to re-name this to scope it to pymevis. May as well make it match convention as well.
```suggestion
pymevis-zoom-factor : default = 1.1, adjusts zoom sensitivity by adjusting magnification factor per scroll event
```
Will require an edit in visCore as well (I should have picked this up earlier)

Reply:
That's fine. I will require an edit in `gl_render3D_shaders` as well. What's the `visCore` edit?